syft/workers/base.py

```diff
@@ -833,8 +842,8 @@ def generate_triple(
         Returns:
             A triple of AdditiveSharedTensors such that c_shared = cmd(a_shared, b_shared).
         """
-        a = self.torch.randint(field, a_size)
-        b = self.torch.randint(field, b_size)
+        a = self.framework.randint(field, a_size)
```

Comment:
does randint behave the same in torch and tf?

Reply:
no, there should be a TODO here.

Reply:
actually, a new PR moved this to the crypto folder, so it's no longer necessary after updating from dev